portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/CKEditor.path

```diff
@@ -153,6 +158,17 @@
 	<td>//div[label[contains(.,'Video')]]//input</td>
 	<td></td>
 </tr>
+<!--SOURCE_CODE_PREVIEW-->
+<tr>
+	<td>CODE_PREVIEW_IFRAME</td>
+	<td>//*[contains(@class,'code_preview')]//iframe</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SOURCE_CODE_DIALOG</td>
+	<td>//*[contains(@class,'cke_dialog_contents')]//*[contains(@class,'CodeMirror cm-s-default')]</td>
```

Comment:
I would caution against having two classes within a contains and split the classes up. What if for instance a developer adds another class that happens to go inbetween the two ie: `CodeMirror NewClass cm-s-default` this will cause an unnecessary test fix. I've seen this recently with https://issues.liferay.com/browse/LRQA-61892

If the two elements are necessary, we would want to split them up like `//*[contains(@class,'cke_dialog_contents')]//*[contains(@class,'CodeMirror') and contains(@class,'cm-s-default')]` 

However, I think we can simply use one unique class in that html element level:
`//*[contains(@class,'cke_dialog_contents')]//*[contains(@class,'CodeMirror-wrap')]`

Reply:
I tested this locally and it works. Will add the commit and forward.